src/test/email.test.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable camelcase */
+/* eslint-env mocha */
```

Comment:
Is there a reason we have these eslint-disable rules at the top here?

Reply:
Removed